internal/action/command.go

```diff
@@ -516,7 +516,11 @@ func SetGlobalOptionNative(option string, nativeValue interface{}) error {
 	// check for local option first...
 	for _, s := range config.LocalSettings {
 		if s == option {
-			MainTab().CurPane().Buf.SetOptionNative(option, nativeValue)
+			if bp, ok := MainTab().CurPane().(*BufPane); ok {
+				bp.Buf.SetOptionNative(option, nativeValue)
+			} else {
+				InfoBar.Error("This pane doesn't support local options")
```

Comment:
I think this error message is gonna be confusing to the user in *any* scenarios of using `SetGlobalOptionNative()`.

I think `SetGlobalOptionNative()` should not display any error message in the infobar on its own, it should rather *return* an error to the caller.

Reply:
Yes, here fully ACK...I've overseen the return type/statement.